### DISCUSSION TITLE
Welcome to budgetbee Discussions!

### DISCUSSION THREAD

**Discussion by Pelukosa** [budgetbee/budgetbee#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FrontierProphet** [comment]
Very excited to see where this project goes!

**Comment by Buzzmovies** [comment]
Is this project still under development?
